components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -2434,7 +2434,7 @@ end subroutine ma_activate_convproc_method2
 !=========================================================================================
    subroutine ma_resuspend_convproc(                           &
               dcondt,  dcondt_resusp,                          &
-              const,   dp_i,          ktop,  kbot_prevap,  pcnst_extd ) ! REASTER 08/05/2015
+              ktop,  kbot_prevap,  pcnst_extd ) ! REASTER 08/05/2015
```

Comment:
We can probably remove the `! REASTER` comment, since it doesn't indicate anything now.